### ISSUE TITLE
Change all Images PNG for  Scalable Vector Graphics(SVG) 

### ISSUE THREAD

**Issue by EvertonMelo** [ethereum/remix-ide#2109]
Hi, 
I took the liberty of listing all the .png images listed in this repository: 
( **78** to 652 files ) 

The sizes are variable but problems like version management and accessibility is something very complicated.

A general overview and SVG applications according to the official website:

Graphics defined with SVG have many different applications. As a result, not all software that uses SVG will have the same features. Conformance to the SVG specification is therefore not a binary matter; sofware may be conforming within a restricted feature set.

Furthermore, SVG is designed to be integrated into other types of documents; depending on the type of integration, only a limited feature-set may be appropriate. There are various ways that an SVG document fragment can be referenced by or included in other documents and thereby be processed by a user agent. SVG documents can also be viewed directly, as the primary document. Each different method by which an SVG document fragment can be used implies a certain set of requirements on how the SVG document fragment must be processed.

other advantages

[5.1. Defining an SVG document fragment: the ‘svg’ element](https://www.w3.org/TR/SVG2/struct.html#NewDocument)
An SVG document fragment consists of any number of SVG elements contained within an ‘svg’ element.
An SVG document fragment can range from an empty fragment (i.e., no content inside of the ‘svg’ element), to a very simple SVG document fragment containing a single SVG graphics element such as a ‘rect’, to a complex, deeply nested collection of container elements and graphics elements.
An SVG document fragment can stand by itself as a self-contained file or resource, in which case the SVG document fragment is an SVG document, or it can be embedded inline as a fragment within a parent HTML or XML document. 
The following example shows simple SVG content embedded inline as a fragment within a parent XML document. Note the use of XML namespaces to indicate that the ‘svg’ and ‘ellipse’ elements belong to the SVG namespace:

[15.5. Hit-testing and processing order for user interface events](https://www.w3.org/TR/SVG2/interact.html#pointer-processing)
There are two distinct aspects of pointer-device interaction with an element or area:
hit-testing, to determine if a pointer event (such as a mouse movement or mouse click) occurred within the interaction area of an element, and the subsequent DOM event flow;
functional processing of actions associated with any relevant element.

[2.4.5. Conforming SVG Stand-Alone Files](https://www.w3.org/TR/SVG2/conform.html#ConformingSVGStandAloneFiles )
A document is a conforming SVG stand-alone file if:
    it is a well-formed XML document,
    its root element is an ‘svg’ element,
    the SVG document fragment rooted at the document element is a conforming XML-compatible SVG markup fragment, and
    any other SVG document fragments within the document (such as those within a ‘foreignObject’) form a conforming XML-compatible SVG markup fragment.

if you convert the files:

│   ├── images
│   │   ├── remix_analysistab.png
│   │   ├── remix_compiletab.png
│   │   ├── remix_debuggertab.png
│   │   ├── remix_editor.png
│   │   ├── remix_file_explorer_browser.png
│   │   ├── remix_file_explorer_menu.png
│   │   ├── remix_quickstart_javascriptvm_callinginstance.png
│   │   ├── remix_quickstart_javascriptvm_creation.png
│   │   ├── remix_quickstart_javascriptvm_creationTransaction.png
│   │   ├── remix_recorder.png
│   │   ├── remix_runtab_example.png
│   │   ├── remix_runtab.png
│   │   ├── remix_settingstab.png
│   │   ├── remix_supporttab.png
│   │   ├── remix_terminal.png
│   │   └── remix_unittest.png
....
│   ├── mist1.png
│   ├── packages.md
│   ├── quickstart_javascript_vm.md
│   ├── remix1.png
│   ├── remix2.png
│   ├── remix3.png
│   ├── remix4.png
│   ├── remix5.png
│   ├── remix_breakpoint.png
│   ├── remixd_alert.png
│   ├── remixd_connectionok.png
│   ├── remix_debuginstructions.png
│   ├── remix_debugtransactioninfo.png
│   ├── remixd_noconnection.png
│   ├── remix_enterdebugsession.png
│   ├── remix_executionexception.png
│   ├── remix_navigation.png
│   ├── remix_plugin_api.md
│   ├── remix_plugin.md
│   ├── remix_soliditylocals.png
│   ├── remix_soliditystate.png
│   ├── remix_startdebugging.png
│   ├── remix_stepdetail.png
│   ├── remix_valueinput.png
...
│   ├── tuto_basicimport.png
│   ├── tuto_importgit.png
│   ├── tuto_importswarm.png
...
│   ├── images
│   │   ├── remix_analysistab.png
│   │   ├── remix_compiletab.png
│   │   ├── remix_debuggertab.png
│   │   ├── remix_editor.png
│   │   ├── remix_file_explorer_browser.png
│   │   ├── remix_file_explorer_menu.png
│   │   ├── remix_quickstart_javascriptvm_callinginstance.png
│   │   ├── remix_quickstart_javascriptvm_creation.png
│   │   ├── remix_quickstart_javascriptvm_creationTransaction.png
│   │   ├── remix_recorder.png
│   │   ├── remix_runtab_example.png
│   │   ├── remix_runtab.png
│   │   ├── remix_settingstab.png
│   │   ├── remix_supporttab.png
│   │   └── remix_terminal.png
...
│   ├── mist1.png
│   ├── quickstart_javascriptvm.rst
│   ├── remix1.png
│   ├── remix2.png
│   ├── remix3.png
│   ├── remix4.png
│   ├── remix5.png
│   ├── remix_breakpoint.png
│   ├── remixd_alert.png
│   ├── remixd_connectionok.png
│   ├── remix_debuginstructions.png
│   ├── remix_debugtransactioninfo.png
│   ├── remixd_noconnection.png
│   ├── remix_enterdebugsession.png
│   ├── remix_executionexception.png
│   ├── remix_navigation.png
│   ├── remix_soliditylocals.png
│   ├── remix_soliditystate.png
│   ├── remix_startdebugging.png
│   ├── remix_stepdetail.png
│   ├── remix_valueinput.png
...
│   ├── tuto_basicimport.png
│   ├── tuto_importgit.png
│   ├── tuto_importswarm.png
..
├── remix-analyzer
│   ├── src
│   │   └── solidity-analyzer
│   │       └── modules
│   │           ├── guardConditions.js.png
...
│   ├── icon.png

for .svg, we will have more agility to allocate and improve performance mainly of I/O
all files when they are .svg will be allocated on the clients, not on the server, and all will be drawn at a faster speed than they will be .xml no more compact images .png

we can greatly improve the performance of this repository for the online service:
https://remix.ethereum.org/

I only need authorization to modify the files and make linkedition. 
I'm willing to listen to other points, as long as they are logical and based on sources.

**Comment by EvertonMelo** [comment]
@yann300 thanks for the feed back.

I'll start the "PullRequests." 

an interesting thing already can send the files with assistive technology tags to the visually impaired using screen readers.

**Comment by yann300** [comment]
@evertonfraga note that all the images you have listed are for the documentation and not for the Remix IDE web app. I don't really think it make sense to spend an importan amount of time for this.
maybe your help us on fixing others issues that are more critical

**Comment by evertonfraga** [comment]
Hi @yann300, it seems you tagged the wrong Everton :)

**Comment by EvertonMelo** [comment]
@yann300  

humm, 
yes you are right, but we can keep the default for visual identity, I will apply tb the same to the remix (only remix not remix-ide ) 
https://github.com/ethereum/remix 

but the observations are still valid for both contexts. 

the repository: https://github.com/ethereum/remix 

the files are:

│   │   ├── remix_analysistab.png
│   │   ├── remix_compiletab.png
│   │   ├── remix_debuggertab.png
│   │   ├── remix_editor.png
│   │   ├── remix_file_explorer_browser.png
│   │   ├── remix_file_explorer_menu.png
│   │   ├── remix_quickstart_javascriptvm_callinginstance.png
│   │   ├── remix_quickstart_javascriptvm_creation.png
│   │   ├── remix_quickstart_javascriptvm_creationTransaction.png
│   │   ├── remix_recorder.png
│   │   ├── remix_runtab_example.png
│   │   ├── remix_runtab.png
│   │   ├── remix_settingstab.png
│   │   ├── remix_supporttab.png
│   │   └── remix_terminal.png
│   ├── mist1.png
│   ├── remix1.png
│   ├── remix2.png
│   ├── remix3.png
│   ├── remix4.png
│   ├── remix5.png
│   ├── remix_breakpoint.png
│   ├── remixd_alert.png
│   ├── remixd_connectionok.png
│   ├── remix_debuginstructions.png
│   ├── remix_debugtransactioninfo.png
│   ├── remixd_noconnection.png
│   ├── remix_enterdebugsession.png
│   ├── remix_executionexception.png
│   ├── remix_navigation.png
│   ├── remix_soliditylocals.png
│   ├── remix_soliditystate.png
│   ├── remix_startdebugging.png
│   ├── remix_stepdetail.png
│   ├── remix_valueinput.png
│   ├── tuto_basicimport.png
│   ├── tuto_importgit.png
│   ├── tuto_importswarm.png
├── filepng.txt

*39* to 404 files

as to whether to be critical or not, I see that performance, better forms of contributions, and assistive technology may be more valid for the project as a whole than the correction of a one-off bug.

**Comment by yann300** [comment]
> as to whether to be critical or not, I see that performance, better forms of contributions, and assistive technology may be more valid for the project as a whole than the correction of a one-off bug.

True, I wanted to let you know that this will not improve remix.ethereum.org. Knowing that, you might not want to push this changes further.
Any kind of contribution is anyway very welcome ;)

**Comment by EvertonMelo** [comment]
@yann300 change status for draft, pl.
